src/data/pokemon/species_info/gen_1_families.h

```diff
@@ -605,6 +623,17 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .iconPalIndex = 0,
         SHADOW(2, 13, SHADOW_SIZE_L)
         FOOTPRINT(Charizard)
+        #if OW_POKEMON_OBJECT_EVENTS && OW_BATTLE_ONLY_FORMS
+            OVERWORLD(
```

Comment:
Please align the `OVERWORLD` macros to `FOOTPRINT`, with the `#if` 4 spaces back.